ext/RMagick/rmimage.c

```diff
@@ -4124,9 +4124,9 @@ Image_constitute(VALUE class ATTRIBUTE_UNUSED, VALUE width_arg, VALUE height_arg
     width = NUM2ULONG(width_arg);
     height = NUM2ULONG(height_arg);
 
-    if (width == 0 || height == 0)
```

Comment:
@dlemstra not sure if there's a better way to check unsigned longs for negative values, but it didn't work without using the `(int)` casts.